uniwersytet-im-adama-mickiewicza-w-poznaniu-wydzial-anglistyki.csl

```diff
@@ -238,9 +238,9 @@
             <text prefix=", " variable="container-title" font-style="italic"/>
             <!-- 2014-02-07 13:11 - changed text prefix from " , " to ", " : CSL 1 does better at localisation and appears to require this workaround precaution no longer: following comma does NOT go under article title's end quote -->
             <group prefix=" ">
-              <text variable="volume"/>
-              <text variable="issue" prefix=", " suffix=""/>
-              <!--Changed Prefix and suffix, PK 2010-02-27-->
+              <text variable="volume" suffix=", "/>
```

Comment:
You're better off using group delimiters rather than these various pre/suffixes.

`        <else>
          <group prefix=" " delimiter=", ">
            <text macro="title" quotes="true"/>
            <group delimiter=" " suffix=".">
              <text variable="container-title" font-style="italic"/>
              <group delimiter=": ">
                <group delimiter=", ">
                  <text variable="volume"/>
                  <text variable="issue" suffix=""/>
                </group>
                <text variable="page"/>
              </group>
            </group>
          </group>
        </else>`

Check this out. It'll work a lot nicer and is also easier to read.

Reply:
Yes, this indeed seems to work :) .
I see that generally it seems preferable to use text variables without prefixes / suffixes and apply these to group delimiters if needed, right?
I have also removed the empty suffix from 
`<text variable="issue" suffix=""/>
`
to 
 `<text variable="issue"/>`

and all seems well in my tests. 

I will be trying this out yet for a bit, and will come back with another commit when ready.

Many thanks!

Reply:
Yeah, exactly.
Yup, that empty suffix is obsolete. Forgot to mention it.